xml/ha_storage_protection.xml

```diff
@@ -842,21 +793,128 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
 
    <para> After the resource has started, your cluster is successfully
     configured for use of SBD and will use this method in case a
-    node needs to be fenced. </para>
+    node needs to be fenced.</para>
+  </sect1>
+
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting Up Diskless SBD</title>
+   <para>SBD can be operated in a diskless mode. In this mode, a watchdog device
+    will be used to reset the node in the following cases: if it loses quorum,
+    if any monitored daemon is lost and not recovered or if Pacemaker decides
```

Comment:
"... if it loses quorum, if and monitored daemon..." sounds weird to me. "or" missing?

Reply:
No, the 'or' was there (a bit later in the sentence), added a comma to make it clearer.